### ISSUE TITLE
Does RussianDoom support FLAC and/or OGG soundtrack?

### ISSUE THREAD

**Issue by vanfanel** [Russian-Doom/russian-doom#159]
Hi there again, @JNechaevsky 

I have tried this pack with russian-heretic:
http://sc55.duke4.net/flac/heretic_sc55_flac.zip
along with the corresponding Cholocate-Doom config:
https://github.com/chocolate-doom/music-packs/blob/master/heretic-music.cfg
(I had to make russian-heretic-setup point to the cfg, of course, which lives in the same directory as the FLAC files).

But I get "Error loading MIDI: Unrecognized audio format" when it tries to load a music track.
So, is FLAC music supported at all?

Thanks!

**Comment by JNechaevsky** [comment]
Hi @vanfanel!

RD does not seems to support music packs in folders, like it's done in Chocolate Doom.
MP3/OGG/FLAC is supported though, if they are placed in PWAD with same name to original lumps (a), and if "MIDI/MP3/OGG/FLAC" was choosen in Setup executable (b).

**Comment by fabiangreffrath** [comment]
It's important that the SDL2_Mixer library supports these formats, else you'll have no chance.

**Comment by vanfanel** [comment]
@JNechaevsky @fabiangreffrath Thanks but it does not seem to work here.
-My SDL2_Mixer lib is built with FLAC and OGG support.
-I have created WADs with FLAC and OGG files named as the original IWAD lumps music files.
For example, for FLAC my WAD has these inside:

```
MUS_CPTD.FLAC 
MUS_E1M3.FLAC
MUS_E1M6.FLAC
MUS_E1M9.FLAC
MUS_E2M3.FLAC
MUS_E2M7.FLAC
MUS_E3M2.FLAC
MUS_TITL.FLAC
MUS_E1M1.FLAC
MUS_E1M4.FLAC
MUS_E1M7.FLAC
MUS_E2M1.FLAC
MUS_E2M4.FLAC
MUS_E2M8.FLAC
MUS_E3M3.FLAC
MUS_E1M2.FLAC
MUS_E1M5.FLAC
MUS_E1M8.FLAC
MUS_E2M2.FLAC
MUS_E2M6.FLAC
MUS_E2M9.FLAC  
MUS_INTR.FLAC

```
I have also tried lowercase names in the WAD, and lowercase/uppercase named OGGs.


-I have selected `MIDI/MP3/OGG/FLAC` in `russian-setup` for Heretic.
I am passing the WAD containing the FLACs or OGGs like this:
`./russian-heretic -pwad heretic-music.wad`
But music is still absent and I see in the console:
`Error loading midi: Unrecognized audio format`

So, any ideas on what could be wrong here?
